File: Smaller Sum[ 22-April-2023]/Brute Force Approach/Solution.cpp
//     >> C++ CODE <<


vector<long long> smallerSum(int n, vector<int>& arr) {
        vector<long long> ans;
        for (int i = 0; i < n; i++) {
            long long cnt = 0;
            for (int j = 0; j < n; j++) {
                if (arr[j] < arr[i]) {
                    cnt += arr[j];
                }
            }
            ans.push_back(cnt);
        }
        return ans;
    }

File: Smaller Sum[ 22-April-2023]/Brute Force Approach/Solution.java
//  >> JAVA CODE 

public long[] smallerSum(int n,int arr[])
    {
        ArrayList<Long> ans = new ArrayList<>();
        for (int i = 0; i < n; i++) {
            long cnt = 0;
            for (int j = 0; j < n; j++) {
                if (arr[j] < arr[i]) {
                    cnt += arr[j];
                }
            }
            ans.add(cnt);
        }
        
            return ans.stream().mapToLong(i->i).toArray();
    }